src/default-theme/variables.css

```diff
@@ -0,0 +1,72 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+
+
+:root {
+
+  /* Borders */
+
+  --jp-borderWidth: 1px;
```

Comment:
The convention we are using is that the last segment is a camel case CSS property, so that includes `borderWidth` and `fontSize` and `color`, but `font-color` isn't one, The `font-` part modifies the `color` CSS property.